### PR TITLE
Created a new function "returnDaysUntil" and neccessary tests

### PR DESCRIPTION
**Pull request by ingmarhendrik**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,8 +1,21 @@
...
-    else {
-        return b;
-    }
-}
\ No newline at end of file
+    return a > b ? a : b;
+};
+
+function returnDaysUntil(targetDate) {
+    console.log('I got there')
```
Comment (ingeliza) [inline index.js:10]: Let's remove console.logs

File: index.js [hunk 2]
```diff
@@ -1,8 +1,21 @@
...
+    const today = moment();
+    const target = moment(targetDate, 'YYYY-MM-DD');
+    console.log("Target", target);
+    console.log("Today",today);
+    const daysDifference = target.diff(today, 'days');
+    console.log(daysDifference);
+    return daysDifference;
+}
+
+// returnDaysUntil('2030-01-01')
```
Comment (ingeliza) [inline index.js:21]: We can remove this comment line as we don't need to run this function right now.

File: index.js (hunk 1 above)
Comment (ingmarhendrik) [reply index.js:10]: why

Comment (ingeliza) [reply index.js:10]: Leaving them in your production code can impact performance and potentially expose sensitive information

Comment (ingmarhendrik) [reply index.js:10]: > Leaving them in your production code can impact performance and potentially expose sensitive information

source: chatgpt

**Review by ingeliza — APPROVED** [review]
Okay tokay :D

**Pull request merged**